otherlibs/win32unix/Makefile

```diff
@@ -34,7 +34,7 @@ UNIX_FILES = access.c addrofstr.c chdir.c chmod.c cst2constr.c \
   exit.c getaddrinfo.c getcwd.c gethost.c gethostname.c \
   getnameinfo.c getproto.c \
   getserv.c gmtime.c mmap_ba.c putenv.c rmdir.c \
-  socketaddr.c strofaddr.c time.c unlink.c fsync.c
+  socketaddr.c strofaddr.c time.c unlink.c fsync.c socketaddr.h
```

Comment:
The C stub files in win32unix are compiled with `-I../unix`, so do we really need to copy `socketaddr.h` from `../unix`?

Reply:
I hadn't registered that, but as it happens I think it is still necessary - it's included in `HEADERS` and so installed and the general rule in `Makefile.otherlibs.common` doesn't know differently.